### DISCUSSION TITLE
Get the apis to execute from csv

### DISCUSSION THREAD

**Discussion by Purushottam-industrility** [artilleryio/artillery#1149] [Artillery]
I am trying to get the list of apis from the csv file. I am expecting that the list of apis should be executed one after another, and when all apis are done, execution of testing should stop and I should get the final result of success and failure count. Can this be done using Artillery?

**Comment by hassy** [comment]
hi @Purushottam-industrility - would something like this work? https://github.com/artilleryio/artillery-examples/tree/main/table-driven-functional-tests
